After Podman version update to 5.5.0, Error: unable to connect to Podman socket: Get "http://d/v5.5.0/libpod/_ping": dial unix /var/run/podman/podman.sock: connect: permission denied: unix:///var/run/podman/podman.sock

```
Run set -Eeuxo pipefail
+ mkdir -p /home/runner/.local/share/containers/tmpdir
+ sudo mount --bind -o rw,noexec,nosuid,nodev,bind /home/runner/.local/share/containers/tmpdir /var/tmp
+ sudo cp ci/cached-builds/containers.conf /etc/containers.conf
+ sudo cp ci/cached-builds/containers.conf /home/linuxbrew/.linuxbrew/opt/podman/etc/containers.conf
+ sudo cp ci/cached-builds/storage.conf /etc/containers/storage.conf
+ sudo cp ci/cached-builds/storage.conf /home/linuxbrew/.linuxbrew/opt/podman/etc/containers/storage.conf
+ sudo cp ci/cached-builds/registries.conf /etc/containers/registries.conf
+ sudo cp ci/cached-builds/registries.conf /home/linuxbrew/.linuxbrew/opt/podman/etc/containers/registries.conf
+ mkdir -p /home/runner/.local/share/containers/storage/tmp
+ sudo /home/linuxbrew/.linuxbrew/opt/podman/bin/podman system reset --force
 A "/home/linuxbrew/.linuxbrew/etc/containers/storage.conf" config file exists.
Remove this file if you did not modify the configuration.
time="2025-05-19T19:19:21Z" level=error msg="shutting down container storage: umount /home/runner/.local/share/containers/storage/overlay, flags: 0x2: no such file or directory"
+ sudo mkdir -p /usr/local/lib/systemd/system/
+ sudo cp ci/cached-builds/podman.service /usr/local/lib/systemd/system/podman.service
+ sudo cp ci/cached-builds/podman.socket /usr/local/lib/systemd/system/podman.socket
+ sudo systemctl daemon-reload
+ sudo systemctl unmask --now podman.service podman.socket
+ sudo systemctl start podman.socket
+ echo PODMAN_SOCK=/var/run/podman/podman.sock
+ podman ps
Cannot connect to Podman. Please verify your connection to the Linux system using `podman system connection list`, or try `podman machine init` and `podman machine start` to manage a new Linux VM
Error: unable to connect to Podman socket: Get "http://d/v5.5.0/libpod/_ping": dial unix /var/run/podman/podman.sock: connect: permission denied: unix:///var/run/podman/podman.sock
Error: Process completed with exit code 125.
```

https://github.com/opendatahub-io/notebooks/actions/runs/15118245091/job/42504167215

--- a/.github/workflows/build-notebooks-TEMPLATE.yaml
+++ b/.github/workflows/build-notebooks-TEMPLATE.yaml
@@ -186,6 +186,10 @@ jobs:
           # remote (CONTAINER_HOST) podman does not do reset (and refuses --force option)
           sudo /home/linuxbrew/.linuxbrew/opt/podman/bin/podman system reset --force
 
+          # https://github.com/containers/podman/pull/25504
+          # podman 5.5.0: The podman system reset command no longer removes the user's podman.sock API socket
+          sudo rm -rf /var/run/podman
+
           # https://github.com/containers/podman/blob/main/docs/tutorials/socket_activation.md
           # since `brew services start podman` is buggy, let's do our own brew-compatible service
           # Regarding directory paths, see https://unix.stackexchange.com/questions/224992/where-do-i-put-my-systemd-unit-file
@@ -204,7 +208,12 @@ jobs:
 
       - name: Show error logs (on failure)
         if: ${{ failure() }}
-        run: journalctl -xe
+        run: |
+          set -Eeuxo pipefail
+
+          journalctl -xe
+          ls -AlF /var/run/podman/podman.sock || echo "Socket /var/run/podman/podman.sock not found"
+          sudo ss -xlpn | grep 'podman.sock' || echo "No active listener found for podman.sock via ss"
 
       - name: Calculate image name and tag
         id: calculated_vars

--- a/ci/cached-builds/podman.service
+++ b/ci/cached-builds/podman.service
@@ -1,3 +1,4 @@
+# https://github.com/containers/podman/blob/main/contrib/systemd/system/podman.service.in
 # https://docs.podman.io/en/latest/markdown/podman-system-service.1.html
 # cat /usr/lib/systemd/system/podman.socket
 

--- a/ci/cached-builds/podman.socket
+++ b/ci/cached-builds/podman.socket
@@ -1,3 +1,4 @@
+# https://github.com/containers/podman/blob/main/contrib/systemd/system/podman.socket
 # cat /usr/lib/systemd/system/podman.socket
 
 [Unit]

